bot/exts/discord/discord.py

```diff
@@ -36,6 +43,49 @@ async def avatar(self, inter: ApplicationCommandInteraction, user: User) -> None
 
         await inter.response.send_message(embed=embed)
 
+    @discord.sub_command()
+    async def server(
+        self,
+        inter: ApplicationCommandInteraction,
+        guild_id: Optional[Guild] = None,
+        invite: Optional[Invite] = None,
+    ) -> None:
+        """
+        Get the icon of a Discord server.
+
+        Parameters
+        ----------
+        guild_id: The ID of the Discord server
+        invite: The invite to the Discord server
+        """
+        match (guild_id, invite):
+            case (None, None):
+                guild = inter.guild
+                icon = guild.icon
+            case (None, invite):
+                if (guild := invite.guild) is None:
+                    raise BadArgument("Cannot fetch the icon of a group DM.")
+                icon = guild.icon
+            case (guild, _):
+                # Only `guild_id` or, `guild_id` and `invite` specified.
+                icon = guild.icon
+
+        if icon is None:
+            embed = create_embed(
+                embed_type="warning",
```

Comment:
```suggestion
                embed_type="error",
```
I think `error` fits better.

Reply:
okay